src/main/scala/mesosphere/marathon/state/Container.scala

```diff
@@ -55,7 +62,8 @@ object Container {
       val UDP = "udp"
 
       val portMappingValidator = validator[PortMapping] { portMapping =>
-        portMapping.protocol is oneOf(TCP, UDP)
+        portMapping.protocol.split(',').each is oneOf(TCP, UDP)
```

Comment:
Perhaps adding a `lazy val protocols = protocol.split(',')` to PortMappings and use that?